bundle/Templating/Twig/BetterIbexaAdminUIExtension.php

```diff
@@ -18,4 +19,14 @@ public function getFunctions(): array
             ),
         ];
     }
+
+    public function getFilters()
+    {
+        return [
+            new TwigFunction(
```

Comment:
You're using a Twig function under filters.  This should be moved to `getFunctions`.